test/hotspot/jtreg/runtime/os/HugePageConfiguration.java

```diff
@@ -86,6 +86,18 @@ public long getThpPageSize() {
         return _thpPageSize;
     }
 
+    public long getThpPageSizeFallback() {
```

Comment:
Can you please slightly rename this to `getThpPageSizeOrFallback`, and write a comment along the lines of:
```
Returns the THP page size; if the kernel does not expose that, return a guessed page size.
Mimics HugePages::thp_pagesize_fallback() in hotspot and must be kept in sync with that function.
```